repository.yaml

```diff
@@ -0,0 +1,7 @@
+templates:
+  - name: TypeScript - Starter
+    path: ./typescript-starter
+  - name: JavaScript - Starter
+    path: ./javascript-starter
+  - name: Golang - Starter
+    path: ./golang-starter
```

Comment:
Looks great!

I would hold off officially releasing this for a bit. I would be good to be able to co-ordinate the announcement and release, so we can post to relevant communities/social as well.

Reply:
I have targeted the wrong branch, should be main not develop :-O